.buildkite/solana-lab.yml

```diff
@@ -0,0 +1,103 @@
+x-default-agents: &default-agents
+  agents:
+    queue: "${AGENT_QUEUE:-default}"
+
+x-test-template: &test-template
+  <<: *default-agents
+  depends_on: ["build"]
+  concurrency: 1
+  concurrency_group: pulumi-testing
+
+  command: |
+    source opsh
+    source .buildkite/git.opsh
+    source .buildkite/lib.opsh
+
+    # setup an ssh agent.  Will need to use '-F $_OPSH_SSH_CONTEXT/config' later for ssh
+    ssh::setup
+
+    # Setup ssh for private repo
+    buildkite-agent secret get GITHUB_TOOLING_KEY | ssh::key::add
+    git::repo-add git@github.com:abklabs/solana-lab.git solana-lab
+    git::repo-add git@github.com:abklabs/tooling.git tooling --ssh-secret GITHUB_TOOLING_KEY
+    git::repos-summary
+    git::repos-clone
+
+    if [[ -n "${SVMKIT_DIR:-}" || -n "${PULUMI_SVMKIT_DIR:-}" ]]; then
+      echo "-- Downloading Pulumi SDK and provider plugin artifacts"
+      buildkite-agent artifact download pulumi-artifacts.tgz .
+      tar zxf pulumi-artifacts.tgz
+      PATH="$PWD/pulumi-svmkit/bin:$PATH"
+      pkg_dir="$$PULUMI_SVMKIT_DIR/sdk/nodejs/bin"
+      yarn link --cwd "$$pkg_dir"
+      
+    fi
+
+    mkdir -p svmkit pulumi-svmkit
+    CLOUD=aws  && . .buildkite/cloud-creds.sh
+    CLOUD=gcp  && . .buildkite/cloud-creds.sh
+    PULUMI_ACCESS_TOKEN=$(buildkite-agent secret get PULUMI_ACCESS_TOKEN)
+
+    # Now that we are here, go to solana-lab:
+    # 1) Install pulumi
+    # 2) Create a stack
+    # 4) run tests
+    # 5) on EXIT be sure and cleanup the stack
+
+    PULUMI_STACK_NAME="abklabs/buildkite-solana-lab"
+    pulumi-cleanup() {
+        pulumi stack select "$$PULUMI_STACK_NAME"
+        pulumi down --yes || true
+        pulumi stack rm --yes ||  true
+    }
+
+    # Add with-*pulumi* scripts to the path
+    PATH=$(realpath ./tooling/bin):$PATH;
+    (
+      cd solana-lab
+      echo "--- Setting up for Test Suite"
+      pulumi stack init "$$PULUMI_STACK_NAME" || true
+      # Set up a trap on exit just in case something goes wrong.
+      trap 'pulumi-cleanup' EXIT
+      pnpm rebuild
+      pnpm install tap@13
```

Comment:
Why do you need to do this if the tap is already in the `package.json`.   We should either move `tap` to a production dependency, or ask to install developer dependencies.